static/styles/settings.css

```diff
@@ -167,6 +167,20 @@ h3,
     cursor: not-allowed;
 }
 
+#deactivate_realm_personal_setting_container {
+    display: inline-block;
+    @media (max-width: 380px) {
```

Comment:
We generally right these media query CSS separately and probably at the end.

Also would be nice if we could use the existing variables defined in `css_variables.js` for the max-width value.

Reply:
have changed the CSS a little to use `mm_min`.